staging/src/k8s.io/kubectl/pkg/cmd/auth/cani.go

```diff
@@ -227,9 +224,10 @@ func (o *CanIOptions) Validate() error {
 				o.warningPrinter.Print(fmt.Sprintf("resource '%s' is not namespace scoped in group '%s'\n", o.Resource.Resource, o.Resource.Group))
 			}
 		}
-		if !isKnownResourceVerb(o.Verb) {
-			o.warningPrinter.Print(fmt.Sprintf("verb '%s' is not a known verb\n", o.Verb))
-		}
+	}
+
+	if !isKnownResourceVerb(o.Verb) {
+		o.warningPrinter.Print(fmt.Sprintf("verb '%s' is not a known verb\n", o.Verb))
 	}
```

Comment:
No, I don't think that we should merge the two warnings messages.

Reply:
/cose 
the PR according the comments.